workers/unity/Assets/Gdk/Core/Systems/SpatialOSSendSystem.cs

```diff
@@ -19,13 +19,34 @@ internal static class Warnings
         private readonly List<int> registeredReplicators = new List<int>();
         private readonly List<int> commandSenders = new List<int>();
 
+        private int ticks = 0;
+        private const int METRIC_TICKS_UPDATE = 200;
+
+        public delegate Worker.Metrics MetricsComputation();
+        private MetricsComputation metricComputedEverySend;
```

Comment:
On second thought, it might be better to split this out into its own system that deals just with metrics.

Reply:
Done.